### ISSUE TITLE
console - improve stats display performance for current month

### ISSUE THREAD

**Issue by fvanderbiest** [georchestra/georchestra#1831]
#1810 improved performances with regards to previous monthes (archive tables are indexed), but the performance issue is still showing up for current month.

On several SDIs, analytics webservices do not respond anymore starting from the 5th day in the month.

**Comment by Vampouille** [comment]
> On several SDIs,

Maybe they miss #1810 updates ?

**Comment by fvanderbiest** [comment]
Yes, I'm gonna ask @fphg

**Comment by MaelREBOUX** [comment]
I think this issue could be closed now we have this -> https://github.com/MaelREBOUX/georchestra_ogc_stats_analyze

**Comment by fvanderbiest** [comment]
Not really, this issue is about improving real time stats with analytics.
But thanks for mentioning this interesting project :-)
